.github/workflows/main.yml

```diff
@@ -113,21 +115,26 @@ jobs:
         shell: bash -l {0}
         run: |
           pip install -r requirements.txt
+          # FIXME: playwright>=1.18.0 fails in testing `test_isinstance` due to the
+          #        'Maximum call stack size exceeded' error
+          pip install "playwright<1.18.0" && python -m playwright install
```

Comment:
Playwright >= 1.18.0 (which comes with chrome >= v99) fails on this recursion test. I'm not sure this is a Playwright issue or a Chrome issue.

https://github.com/python/cpython/blob/0a7dcbdb13f1f2ab6e76e1cff47e80fb263f5da0/Lib/test/test_isinstance.py#L274-L282